bylaws/001-mission-and-structure.md

```diff
@@ -28,6 +28,14 @@ If, in the judgement of the PHP FIG, a Project Representative is acting inapprop
 
 A Project Representative may also simultaneously serve as a Core Committee member, PSR Editor or Working Group member but not as a Secretary.
 
+## Conduct
+
+All individuals involved with FIG either in an official capacity (Secretary, Core Committee, Project Representative, Editor, or Working Group member) or informally (such as participating in discussions) are expected to conduct themselves in a professional manner at all times.  All individuals are expected to behave in a manner consistent with the Code Manifesto.  Failure to do so will be considered grounds for expulsion.
+
+In the case of an individual holding a formal position, a Recall or Expulsion Vote may be called to remove the individual in question.  In the case of any other individual, any two Secretaries may issue a formal warning to the individual in question or issue a temporary or permanent ban from all FIG-related communication channels, as appropriate to the situation.
```

Comment:
I don't understand "any two Secretaries may issue a formal warning", does it mean that two secretaries have to issue a formal warning, or that one of them does it?

Reply:
I think the meaning is "at least two out of three"

Reply:
Yes, the intent here is that "at least 2 out of 3" can warn/ban/whatever someone.  Unanimity is too high a bar for many cases, but we also don't want any one Secretary to be able to just ban someone unilaterally.  Naturally if all 3 agree that's even better.

I'm mainly thinking here of some rando on the list going off on someone in an obviously inappropriate manner; one secretary says to another secretary "we kick?"  Other secretary responds "yup, kick".  Then rando gets kicked.